### PR TITLE
Create sparse_matrix_addition.c

### PR DESCRIPTION
**Pull request by kiransbaliga**
a program that converts matix to sparse form,find its transpose and addition of the two matrices in sparse form.
*contributed by*
@kiransbaliga[https://github.com/kiransbaliga]

### CODE REVIEW

**Review by GauravWalia19 — CHANGES_REQUESTED** [review]
Please update the README with algorithm you are adding

**Comment by kiransbaliga** [comment]
okay will do, I am new at github,
 so should I do pull request again?

**Review by GauravWalia19 — CHANGES_REQUESTED** [review]
Please remove the MATRIX headings in the README and add the algorithm link under MISC heading

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ For contribution you can see our [guidelines](CONTRIBUTING.md)
 * SEARCHING
 * SORTING
 * MISC
+* MATRIX
```
Comment (GauravWalia19) [inline README.md:14]: Remove this line

**Review by GauravWalia19 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kiransbaliga) [reply README.md]: I have removed it please check it!!

**Review by GauravWalia19 — APPROVED** [review]

**Pull request merged**